pkg/k8sutil/stunnel.go

```diff
@@ -114,6 +115,32 @@ func InsertStunnel(
 			},
 		},
 	})
+
+
+	// Default values for stunnel request and memory
+	// stunnel has been observed to consume between 3 and 6 MB, so
+	// set memory request to 5 and limit to 25.
+	// Also limit cpu usage to 1 whole CPU
+	// In some cases we have seen the memory limit to go higher > 50Mi
+	// these cases can be handled by the configuration variables
+	var limitCPU string = getEnvVarWithDefault("STUNNEL_LIMIT_CPU", "")
```

Comment:
more idiomatic: `limitCPU := getEnvVarWithDefault("STUNNEL_LIMIT_CPU", "")`
same comment for next 2 lines

Reply:
ok, refreshing

Reply:
Updated